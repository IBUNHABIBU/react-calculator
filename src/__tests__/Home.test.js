import Home from '../Components/Home';
import { shallow } from 'enzyme';

let wrapper;
beforeEach(()=> {
    wrapper = shallow(<Home />)
})

describe("Show the home page element", () => {
    it("should display the Home page paragraphs", () => {
        expect(wrapper.find('h1').text()).toContain('Welcome to Math-magician Calculator');
    })
})