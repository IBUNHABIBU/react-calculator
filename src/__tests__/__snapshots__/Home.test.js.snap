// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots renders App component correctly 1`] = `
<div
  className="home-container"
>
  <h1
    align="center"
  >
    Welcome to Math-magician Calculator
  </h1>
  <p>
    In this project you will learn how to build a very simple React application. You will work in a real-world project for a fictional company that requires your help
  </p>
  <p>
    Math-magicians is a fictional company based in France. They have requested your help to build a simple product that would assist them with their calculations. An important thing to notice is that they have asked for a product that is just fully-functional and it does not need to be beautiful and user-friendly (at least in its first version).
  </p>
</div>
`;
